benchmark/atoi-catch2-tests.cpp

```diff
@@ -51,3 +52,8 @@ TEST_CASE("Atol", "[pure-test][swar][atoi]") {
         REQUIRE(converted == expected);
     }
 }
+
+TEST_CASE("String function regressions", "[pure-test][atoi][swar]") {
+    auto input = "\x80\0";
+    REQUIRE(1 == zoo::c_strLength(input));
+}
```

Comment:
This is a regression function: something that early in development was a bug.  I had a broken method that would allow specifically the `char 0x80` to be mis-identified as a null.
Regressions are ad-hoc.  They don't have to be systematic.
If enough to resolve, please 'resolve conversation'